I think the test failure is unrelated to these changes:
```
=========================== short test summary info ============================
FAILED tests/components/generic/test_camera.py::test_fetching_url - AssertionError: assert 500 == <HTTPStatus.OK: 200>
 +  where 500 = <ClientResponse(http://127.0.0.1:42901/api/camera_proxy/camera.config_test) [500 Internal Server Error]>\n<CIMultiDictP...': '4367', 'Date': 'Tue, 30 Jul 2024 10:02:12 GMT', 'Server': 'Python/3.12 aiohttp/3.10.0rc0', 'Connection': 'close')>\n.status
 +  and   <HTTPStatus.OK: 200> = HTTPStatus.OK
FAILED tests/components/generic/test_camera.py::test_limit_refetch - AssertionError: assert 500 == <HTTPStatus.OK: 200>
 +  where 500 = <ClientResponse(http://127.0.0.1:46823/api/camera_proxy/camera.config_test) [500 Internal Server Error]>\n<CIMultiDictP...': '4367', 'Date': 'Tue, 30 Jul 2024 10:02:13 GMT', 'Server': 'Python/3.12 aiohttp/3.10.0rc0', 'Connection': 'close')>\n.status
 +  and   <HTTPStatus.OK: 200> = HTTPStatus.OK
Error: Process completed with exit code 1.
```

--- a/script/install_integration_requirements.py
+++ b/script/install_integration_requirements.py
@@ -45,6 +45,7 @@ def main() -> int | None:
         cmd,
         check=True,
     )
+    return None
 
 
 if __name__ == "__main__":

